<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeGeneratorApp</class>
 <widget class="QMainWindow" name="CodeGeneratorApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PromptCodeLoop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="prompt_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the Prompt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prompt_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>760</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Code</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>760</width>
      <height>300</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="test_case_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Case File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_case_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
